apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -75,8 +75,24 @@ function Navbar({ identity }: NavbarProps) {
 					<NavLinkItem href="/">Schedule</NavLinkItem>
 					<NavLinkItem href="/">Resources</NavLinkItem>
 					<NavLinkItem href="/">Stage</NavLinkItem> */}
+					<NavLinkItem
+						href="http://tinyurl.com/ih24-incident-form"
```

Comment:
Could we add named redirects for these, see #49 for reference.
Also we could use the actual Google Form short URL (`forms.gle/...`) instead of sending this through TinyURL.

Reply:
I could not find the short form of forms.gle, but the full link for docs.google.com/forms does exist